### ISSUE TITLE
Performance Comparison between official Notion and notion-enhanced

### ISSUE THREAD

**Issue by HsunGong** [notion-enhancer/desktop#745]
This page: https://3perf.com/blog/notion/ analyze the performance of notion app, but I wonder how to compare notion-enhanced app with the official one.

Related:
- https://www.notion.so/blog/faster-page-load-navigation
- https://3perf.com/blog/notion/
- https://www.reddit.com/r/Notion/comments/d2us1p/whats_faster_the_app_or_the_browser/
- https://www.reddit.com/r/Notion/comments/cdp293/is_there_any_way_to_speed_up_notion_it_feels/
- https://github.com/NotionX/react-notion-x

**Comment by dragonwocky** [comment]
Sorry, I've only just seen this! There's a lot of good suggestions in that article for improving Notion's performance, but as far as comparing the notion-enhancer with vanilla Notion's performance:
- the availability of Notion's desktop-specific performance improvements depends on the build available from https://github.com/notion-enhancer/notion-repackaged (e.g. the SQLite database is implemented, but icon caching isn't) - when the notion-enhancer is next updated (I'm working on it!) then notion-repackaged will catch up to the latest version of the Notion desktop app and benefit from all those optimisations.
- the web client/interface's baseline performance will be the same, but the notion-enhancer's theming system and certain mods (e.g. the code lines extension) will put additional load on the renderer that can slow Notion down.

I'm always trying to improve the performance of the notion-enhancer with each update, but sometimes there's only so much I can do, since the notion-enhancer essentially has to battle Notion to control the behaviour or appearance of the interface. There's a breakdown of how the notion-enhancer works [on this page](https://notion-enhancer.github.io/documentation/concepts) that goes into a bit more detail on when and how the notion-enhancer performs certain tasks and how they can affect the performance.

There is also a memory leak somewhere within either Notion or the notion-enhancer that I am yet to reliably reproduce or find a cause/fix for. Personally, I haven't experienced it in months, but it's been reported by other users.

Happy to chat more [in the Discord server](https://discord.gg/sFWPXtA), but closing this issue here since it's not exactly a bug report or feature request.

TLDR: notion-enhancer will always be slower than vanilla Notion, but how much slower varies wildly depending on a whole lot of things.
